main.js

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+
+const { run } = require('./');
+
+const argv = require('yargs').command('* <baseSpec> [specs...]', '', argv => {
```

Comment:
No need to move `yargs` statement to a separate file. Just wrap it in 
```js
if (require.main === module) {
 ...
}
```

in the main file.